tests/sentry/utils/email/test_message_builder.py

```diff
@@ -115,8 +84,10 @@ def test_with_users(self):
         user_b = User.objects.create(email="bar@example.com")
         user_c = User.objects.create(email="baz@example.com")
 
+        alternate_email = "bazzer@example.com"
+        UserEmail.objects.create(user=user_c, email=alternate_email)
```

Comment:
I had to add this line to get this test to pass. This is somewhat troubling because I didn't change any functionality. Was the previous filename (`tests/sentry/utils/email/tests.py`) not caught up in tests?

Reply:
🤔 this test is failing on master for me:

```tests/sentry/utils/email/tests.py:128: in test_with_users
    assert len(mail.outbox) == 3
E   AssertionError: assert 2 == 3
E    +  where 2 = len([<django.core.mail.message.EmailMultiAlternatives object at 0x111342da0>, <django.core.mail.message.EmailMultiAlternatives object at 0x111342a58>])
E    +    where [<django.core.mail.message.EmailMultiAlternatives object at 0x111342da0>, <django.core.mail.message.EmailMultiAlternatives object at 0x111342a58>] = mail.outbox```